acceptance/kubernetes/kubernetes.go

```diff
@@ -269,6 +272,8 @@ func AddStepsTo(sc *godog.ScenarioContext) {
 
 func InitializeSuite(ctx context.Context, tsc *godog.TestSuiteContext) {
 	tsc.AfterSuite(func() {
-		kind.Destroy(ctx)
+		if !persisted {
```

Comment:
So we don't need the global variable

```suggestion
		if !testenv.Persisted(ctx) {
```

Reply:
I don't think this instance of context will have the right info. This is the "default" context. Let me try it.

Reply:
It worked! Thank you.